base/docs/basedocs.jl

```diff
@@ -1049,13 +1049,28 @@ exception object to the given variable within the `catch` block.
 
 The power of the `try`/`catch` construct lies in the ability to unwind a deeply
 nested computation immediately to a much higher level in the stack of calling functions.
+
+A `try` or `try`/`catch` block can also have a [`finally`](@ref) clause that executes
+at the end, regardless of whether an exception occurred.  For example, this can be
+used to guarantee that an opened file is closed:
+```julia
+f = open("file")
+try
+    operate_on_file(f)
+catch
+    println("An error occurred!")
```

Comment:
```suggestion
    @warn "An error occurred!"
```
since the earlier example used `@warn`